### ISSUE TITLE
Rename package to `scala.xml` instead of `dotty.xml`

### ISSUE THREAD

**Issue by hamzaremmal** [lampepfl/xml-interpolator#53]

**Comment by nicolasstucki** [comment]
This may cause a conflict when we import `scala.xml` because of the package file we define with the interpolator. Maybe we need to wait for #54.

**Comment by hamzaremmal** [comment]
Yes, I just opened the issue to remember to do it (and so I can also erase it from my board)
